lib/b-tree.js

```diff
@@ -0,0 +1,318 @@
+'use strict';
+
+const DEFAULT_DEGREE = 6; // min degree of b-tree.
+// All vertices except the root have [degree ... 2 * degree] child nodes
+
+function BTree(degree = DEFAULT_DEGREE) {
+  this.root = new BTreeNode();
+  this.minDegree = degree;
+}
+
+function BTreeNode(leaf = true) {
```

Comment:
No need to add BTree, this is already scoped inside BTree file. So just `Node` is fine.
Also same as for Btree, use es6 classes, please.